tests/sync/session.cpp

```diff
@@ -138,10 +142,10 @@ TEST_CASE("SyncSession: management by SyncUser", "[sync]") {
     SECTION("a SyncUser properly rebinds existing sessions upon logging back in") {
         const std::string user_id = "user1d";
         auto user = SyncManager::shared().get_user(user_id, "not_a_real_token");
-        auto session1 = sync_session(server, user, "/test1b-1",
+        auto session1 = sync_session(server, user, "/test1d-1",
                                      [&](auto...) { return s_test_token; },
                                      [&](auto...) { });
-        auto session2 = sync_session(server, user, "/test1b-2",
+        auto session2 = sync_session(server, user, "/test1d-2",
                                      [&](auto...) { return s_test_token; },
```

Comment:
Using `auto...` like this isn't compatible with GCC 4.9. CI doesn't catch this as it does not yet run sync tests.